shellnotes.sh

```diff
@@ -1,159 +1,150 @@
-#remember to write this line into your terminal
-#chmod +x shellnotes.sh
-
+# Change the first line if you don't use bash, in order for the code to work properly.
+# Warning! If you don't have gedit(text editor) or nano installed, you may have to change the default text editor (lines 10, 25, 40).
+# remember to write this line into your terminal
+# chmod +x shellnotes.sh
 
 #Make a Notes folder for the user
 cd ~
 if [[ -d "Notes" ]]; then
-	exists=True;
+	exists=True
 else
-	mkdir Notes;
+	mkdir Notes
 fi
 
-
-
 #opening a note (It will create a new note if $notename is blank)
-function readnote() { 
-dir="$(pwd)";
-me="$(whoami)";
-read -p "Enter note name: " opennote; 
-cd ~/Notes;
-if [ -e $opennote ]; then
-	gedit $opennote;
-	
-else
-	clear;
-	echo "No such note.";
-	cd ~
-	read -p "Do you want to create one?[Y/N]: " create;
+function readnote() {
+	dir="$(pwd)"
+	me="$(whoami)"
+	read -p "Enter note name: " opennote
+	cd ~/Notes
+	if [ -e $opennote ]; then
+		gedit $opennote
+	else
+		clear
+		echo "No such note."
+		cd ~
+		read -p "Do you want to create one?[Y/N]: " create
+
+	fi
 	
-fi
 	if [ $create == "y" ] || [ $create == "Y" ] || [ $create == "yes" ] || [ $create == "YES" ] || [ $create == "Yes" ]; then
 		newnote
 	fi
-	
-clear;
-cd $dir
+
+	clear
+	cd $dir
 
 }
 
 #Read notes instantly via terminal
 function quickread() {
-	dir="$(pwd)";
-	read -p "Enter note name: " quicknotename;
+	dir="$(pwd)"
+	read -p "Enter note name: " quicknotename
 	cd ~/Notes
 	if [ -e $quicknotename ]; then
-		clear;
+		clear
 		cat $quicknotename
-	
 	else
-		clear;
-		echo "No such note.";
+		clear
+		echo "No such note."
 		cd ~
-		
-fi
-
-cd $dir
+	fi
 
+	cd $dir
 }
 
 #A quick solution to users who want keeping notes instantly via terminal.
 function quicknote() {
-dir="$(pwd)"; 
-me="$(whoami)";
-cd ~/Notes;
-read -p "Enter note name: " notename;
-if [ -e $notename ]; then
-	clear;
-	echo "This note already exists.";
-	cd ..
-	read -p "Do you want to read it?[Y/N]: " readquicknote;
-	
-else
-	touch $notename; 
-	nano $notename; 
-	cd $dir;
-	clear;
-	echo "Note created in Home/$me/Notes"; 
-	echo "-----------------------------------"; 
-		
-fi
-
-if [[ $readquicknote == "y" ]] || [[ $readquicknote == "Y" ]] || [[ $readquicknote == "yes" ]] || [[ $readquicknote == "YES" ]] || [[ $readquicknote == "Yes" ]]; then
-	clear;
+	dir="$(pwd)"
+	me="$(whoami)"
 	cd ~/Notes
-	cat $notename;
-	cd $dir
-fi
-}
+	read -p "Enter note name: " notename
+	if [ -e $notename ]; then
+		clear
+		echo "This note already exists."
+		cd ..
+		read -p "Do you want to read it?[Y/N]: " readquicknote
+	else
+		touch $notename
+		nano $notename
+		cd $dir
+		clear
+		echo "Note created in Home/$me/Notes"
+		echo "-----------------------------------"
+	fi
 
+	if [[ $readquicknote == "y" ]] || [[ $readquicknote == "Y" ]] || [[ $readquicknote == "yes" ]] || [[ $readquicknote == "YES" ]] || [[ $readquicknote == "Yes" ]]; then
```

Comment:
The [`case` statement ](https://linuxize.com/post/bash-case-statement/)can help here

```
case $readquicknote in y|Y|yes|YES|Yes ) 
    clear
    cd ~/Notes
    cat $notename
    cd $dir
    ;;
esac